### PR TITLE
Fix for issue caused by new My Profile link

### PR DESCRIPTION
**Pull request by grabartley**

### CODE REVIEW

File: test/specs/profiles/visibility-agg.js [hunk 1]
```diff
@@ -77,7 +77,8 @@ describe('Test profile visibility', function () {
         console.log('user', user);
         promises.push(() => login(user));
         promises.push(() => MainPage.userMenu.click());
-        promises.push(() => MainPage.userMenu_myProfile.click());
+        promises.push(() => MainPage.userMenu_viewProfileLink);
+        promises.push((url) => browser.url(url));
         promises.push(() => browser.getUrl());
```
Comment (Wardormeur) [inline test/specs/profiles/visibility-agg.js]: you already have the url now :)

Comment (grabartley) [inline test/specs/profiles/visibility-agg.js]: Is it passed down by browser.url(url)?

Comment (Wardormeur) [inline test/specs/profiles/visibility-agg.js]: Add return Promise.resolve(url) and get it through the params as line 83, then delete line 82

Comment (grabartley) [inline test/specs/profiles/visibility-agg.js]: Done :)

**Pull request merged**